static/js/biomedical/disease/page.tsx

```diff
@@ -87,7 +92,15 @@ export class Page extends React.Component<PagePropType, PageStateType> {
       { id: "name", name: "Compound Name" },
       { id: "drugSource", name: "Drug Source" },
     ];
-
+    const diseasePrevalenceStatVarValue =
```

Comment:
nit: since this is the stat var dcid, maybe `diseasePrevalenceStatVarDcid` instead

Reply:
Done!